src/blog/2024/003-2023-year-in-review.md

```diff
@@ -0,0 +1,98 @@
+---
+
+title: 2023 in Review
+layout: blogpost.hbs
+
+slug: 2023-year-in-review
+author: Nico Rehwaldt <https://github.com/nikku>
+description: "From custom form components to bpmn-js type declarations, between intelligent FEEL editing and core modeling UX improvements: This post recaps 2023, a great year for bpmn.io."
+preview_image: assets/attachments/blog/2024/003-preview.png
+url: blog/posts/2023-year-in-review.html
+published: 2024-01-24 13:28
+published_time: "2024-01-24T13:28:09+01:00"
+modified_time: "2024-01-24T13:28:09+01:00"
+
+releases:
+
+---
+
+
+<p class="introduction">
+  From custom form components to bpmn-js type declarations, between intelligent FEEL editing and core modeling UX improvements, 2023 was a busy, good year for the [bpmn.io project](https://bpmn.io/) and friends of our editing toolkits. Let us look back at what we accomplished and take a glimpse into the near future, too.
```

Comment:
"2023 was a busy, good year" doesn't sound right to me. Some suggestions to improve the flow

> From the introduction of custom form components to enhanced bpmn-js type declarations, and from advancements in intelligent FEEL editing to significant upgrades in core modeling UX, 2023 was indeed a busy and fruitful year for the [bpmn.io project](https://bpmn.io/) and our community. Let's take a moment to reflect on our achievements and glimpse into the near future.

Reply:
Thanks, updated!